src/NServiceBus.RavenDB/SessionManagement/RavenDbStorageSession.cs

```diff
@@ -27,10 +32,20 @@ protected override void Setup(FeatureConfigurationContext context)
                     var store = DocumentStoreManager.GetDocumentStore<StorageType.Sagas>(context.Settings, b);
                     var storeWrapper = new DocumentStoreWrapper(store);
 
-                    var dbNameConvention = context.Settings.GetOrDefault<Func<IDictionary<string, string>, string>>("RavenDB.SetMessageToDatabaseMappingConvention");
+                    var dbNameConvention = context.Settings.GetOrDefault<Func<IDictionary<string, string>, string>>(SetMessageToDatabaseMappingConventionKey);
                     return new OpenRavenSessionByDatabaseName(storeWrapper, dbNameConvention);
                 }, DependencyLifecycle.SingleInstance);
+
+                context.Settings.AddStartupDiagnosticsSection(StartupDiagnosticsSectionName, new
+                {
+                    SharedSession = false,
+                    CustomDatabaseMappingConvention = context.Settings.HasSetting(SetMessageToDatabaseMappingConventionKey)
+                });
             }
         }
+
+        const string StartupDiagnosticsSectionName = "NServiceBus.Persistence.RavenDB.Session";
```

Comment:
Was on the fence between `NServiceBus.Persistence.RavenDB.Session` and `NServiceBus.Persistence.RavenDB.Sessions`